.infrastructure/08_ecr.tf

```
@@ -0,0 +1,30 @@
+resource "aws_ecr_repository" "strapi_image_repository" {
+  name = "strapi"
```

```suggestion
  name = "strapi"
  
  image_tag_mutability = "IMMUTABLE"
```
the RFC about what container registry to use says that the AWS ECR has this feature, that led us to evaluate the ECR option `above` compared to the alternatives.

should we enable it?

---

yep, indeed the preferred way to deploy images on the registry is to use a unique identifier (like a commit SHA or a tag), that's why in the [PR related to the workflow](https://github.com/pagopa/b2b-portals/pull/33#discussion_r1387801005) I proposed to **not** use the `latest` as tag

---

ok, i will set the tag in the workflow with "${{ github.sha }}" value